src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -0,0 +1,184 @@
+import ConditionalRender from "components/ConditionalRender"
+import DateTime from "components/DateTime"
+import { GridRow, Link, Table } from "govuk-react"
+import Image from "next/image"
+import { useRouter } from "next/router"
+import { encode } from "querystring"
+import { useState } from "react"
+import { createUseStyles } from "react-jss"
+import CourtCase from "services/entities/CourtCase"
+import User from "services/entities/User"
+import { deleteQueryParamsByName } from "utils/deleteQueryParam"
+import getTriggerWithDescription from "utils/formatReasons/getTriggerWithDescription"
+import groupErrorsFromReport from "utils/formatReasons/groupErrorsFromReport"
+import { displayedDateFormat } from "utils/formattedDate"
+import {
+  filterUserNotes,
+  first100CharsOfMostRecentNote,
+  getMostRecentNote,
+  validatedMostRecentNoteDate
+} from "./CourtCaseListEntryHelperFunction"
+import { NotePreview, NotePreviewButton } from "./NotePreviewButton"
+import LockedByTag from "./tags/LockedByTag"
+import ResolvedTag from "./tags/ResolvedTag"
+import UrgentTag from "./tags/UrgentTag"
+
+const useStyles = createUseStyles({
+  caseDetailsRow: {
+    verticalAlign: "top",
+    borderColor: "unset"
+  },
+  triggersRow: {
+    verticalAlign: "top",
+    backgroundColor: "#f3f2f1" // GDS light-grey color
```

Comment:
`#f3f2f1` is used 2x across the codebase. We could have it as a variable in `src/utils/colours.ts`. We already have a few colours in there.

Reply:
This has not been changed, we just broke it out into a new component to simplify `CourtCaseList` and fix an error we were having of using `useState()` within a loop.